lexerState.go

```diff
@@ -57,6 +58,7 @@ var validLexerStates = []lexerState{
 			COMPARATOR,
 			LOGICALOP,
 			CLAUSE_CLOSE,
+			TERNARY,
```

Comment:
Hmm, then I must be doing something wrong. Because if drop for instance string transition this expression fails: `partner == 'amazon' ? 10` with `Cannot transition token types from STRING [amazon] to TERNARY [?]`